Makefile

```diff
@@ -15,9 +15,6 @@ init: init-submodules
 
 update-aws-lc-fips-sys:
 	git submodule update --init --remote --checkout -- aws-lc-fips-sys/aws-lc
-	cd aws-lc-fips-sys/aws-lc && \
-		git fetch --all && \
-		git tag -l | xargs ../../scripts/tools/semver.rs fips-v3 | xargs git checkout
```

Comment:
Why is this removed? Don't we want to stay tracking the FIPS v3 release?

Reply:
Line 17 puts us on the latest commit on the branch. The lines I removed were checking out the latest tag on the branch, which we don't want.